tools/generator/InterfaceGen.cs

```diff
@@ -349,6 +349,7 @@ void GenerateEventArgs (Method m, StreamWriter sw, string indent, CodeGeneration
 			string args_name = GetArgsName (m);
 			if (m.RetVal.IsVoid || m.IsEventHandlerWithHandledProperty) {
 				if (!m.IsSimpleEventHandler || m.IsEventHandlerWithHandledProperty) {
+					sw.WriteLine ("{0}// event args for {1}.{2}", indent, this.JavaName, m.JavaName);
```

Comment:
This line is causing the unit tests to fail, as the expected file contents don't contain it.